app/views/projects/index.html.haml

```diff
@@ -25,8 +26,9 @@
               %br
               %nobr
                 %small
-                  forked from
+                  Forked from:
                   = link_to project.source_full_name, project.source_github_url, target: '_blank'
+          %td= project.language
```

Comment:
the programming language.

I did not chose the variable name, it was already available in code.
I know, the name could be better, but since the project = "a code library", project.language is ok as soon as you realize what a project is.